server/README.md

```diff
@@ -6,6 +6,6 @@ Pick the language you are most comfortable in and follow the instructions in the
 # Supported languages
 
 * [JavaScript (Node)](node/README.md)
+* [PHP (Slim)](php/README.md)
 * [Python (Flask)](python/README.md)
-* [Ruby (Sinatra)](ruby/README.md)
```

Comment:
Is this for alphabetical order? I like it!

Reply:
Yup, that was the intention